src/components/Input/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import * as style from './styles';
+import { CheckBoxProps } from './CheckBoxProps';
+
+const CheckBox = (
+    props: React.PropsWithChildren<CheckBoxProps>
+): JSX.Element => {
+    const className = `${String(style.CheckBoxContainerStyle)}`;
```

Comment:
```suggestion
    const className = `${String(CheckBoxContainerStyle)}`;
```

better to destructure